### PR TITLE
feat(cdk): implement `tuiIsPresent` pipe

### PR DESCRIPTION
**Pull request by theorlovsky**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo/src/modules/pipes/is-present/examples/1/component.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import {tuiIsPresent} from '@taiga-ui/cdk';
+
+@Component({
+    selector: `tui-is-present-example1`,
+    templateUrl: `./template.html`,
+    styleUrls: [`./style.less`],
+    changeDetection,
+    encapsulation,
+})
+export class TuiIsPresentExample1 {
```
Comment (splincode) [inline projects/demo/src/modules/pipes/is-present/examples/1/component.ts:14]: not sure I understand this example

Comment (theorlovsky) [reply projects/demo/src/modules/pipes/is-present/examples/1/component.ts:14]: it's basically a copy-paste from the existing [`tuiIsPresent`](https://github.com/Tinkoff/taiga-ui/blob/main/projects/demo/src/modules/utils/miscellaneous/examples/5/index.ts) function example

Comment (splincode) [reply projects/demo/src/modules/pipes/is-present/examples/1/component.ts:14]: but why should we show how the pipe works in this way?

instead

```html
{{ value | tuiPresent }} = value | tuiPresent
```

Comment (theorlovsky) [reply projects/demo/src/modules/pipes/is-present/examples/1/component.ts:14]: oh, I see. you're right

Comment (theorlovsky) [reply projects/demo/src/modules/pipes/is-present/examples/1/component.ts:13]: updated

**Review by splincode — APPROVED** [review]

**Comment by splincode** [comment]
@nsbarsukov @vladimirpotekhin

**Comment by waterplea** [comment]
I think a nice example would be something like:
```html
<div *ngIf="count$ | async | tuiIsPresent as value">
```
showing how this can be used for falsy values like `0`, but not for `null` while stream has not returned any value.

File: projects/demo/src/modules/pipes/is-present/examples/1/component.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    readonly items = [`String`, `null`, `undefined`];
+    readonly control = new FormControl(null);
+
+    get value(): string | null | undefined {
+        return this.objectifyValue(this.control.value ?? ``);
+    }
+
+    private objectifyValue(value: string): string | null | undefined {
+        if (value === `null`) {
```
Comment (waterplea) [inline projects/demo/src/modules/pipes/is-present/examples/1/component.ts:22]: nit: Looks like a job for `switch` :)

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/pipes/is-present/examples/1/component.ts (hunk 2 above)
Comment (theorlovsky) [reply projects/demo/src/modules/pipes/is-present/examples/1/component.ts:22]: do you want to keep this example or completely replace it with the one above?

Comment (theorlovsky) [reply projects/demo/src/modules/pipes/is-present/examples/1/component.ts]: I went ahead and replaced it since the new example is more useful 🙂

Comment (theorlovsky) [reply projects/demo/src/modules/pipes/is-present/examples/1/component.ts]: @waterplea please check if it's not too complicated

**Review by splincode — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/pipes/is-present/examples/1/component.ts (hunk 2 above)
Comment (waterplea) [reply projects/demo/src/modules/pipes/is-present/examples/1/component.ts]: Looks great. Thank you!

**Pull request merged**